server.js

```diff
@@ -0,0 +1,23 @@
+const Hapi = require("hapi");
+const jsonReturningController = require("./controllers/jsonReturningController");
```

Comment:
In Hapi we didn't using controllers. In most cases in node.js we using route handlers.